Common/UnitDefinitions/Volume.json

```diff
@@ -189,6 +222,9 @@
     {
       "SingularName": "ImperialGallon",
       "PluralName": "ImperialGallons",
+      "BaseUnits": {
+        "L": "Decimeter"
```

Comment:
Will this throw an exception due to multiple units having the same base unit? 
`new Volume(1, new UnitSystem(LengthUnit.Decimeter);`

I wonder if it would be acceptable to just choose one in that case? I'm too far removed from the usecase to know what is expected behavior.

Reply:
It would. We should add a method to return all unit infos that are a subset of baseunits. We could pick the first one.